helm/prometheus-meta-operator/ci/default-values.yaml

```diff
@@ -4,6 +4,8 @@ Installation:
     Monitoring:
       Prometheus:
         Host: ""
+    Provider:
```

Comment:
I am not sure we actually need that kind as apps in tenant cluster already have access to the provider https://github.com/giantswarm/nginx-ingress-controller-app/blob/e44271c88932c1a991a3fea2a90ecd68018c58fd/helm/nginx-ingress-controller-app/values.yaml#L338
Where is that information obtained from?

Reply:
The value for this comes from the installation repo
e.g. https://github.com/giantswarm/installations/blob/master/asgard/draughtsman-configmap-values.yaml#L60

Reply:
:rocket: